.circleci/src/commands/@mobile-commands.yml

```diff
@@ -66,9 +66,12 @@ mobile-prepare-ios:
         command: |
           curl -sL https://sentry.io/get-cli/ | bash
           echo export SENTRY_BINARY=/usr/local/bin/sentry-cli >> "$BASH_ENV"
+    - run:
+        name: Build dependencies
+        command: npx turbo run build --filter=audius-mobile-client
```

Comment:
For mobile I opted to manually run the dependency builds before fastlane because wrapping the fastlane commands as npm scripts requires a lot of argument munging